packages/mobile/src/recipients/recipient.ts

```diff
@@ -204,30 +204,55 @@ const fuzzysortToRecipients = (
   return result
 }
 
-const nameCompare = (a: FuzzyRecipient, b: FuzzyRecipient) => {
+function nameCompare(a: FuzzyRecipient, b: FuzzyRecipient) {
   const nameA = a.displayName.toUpperCase()
   const nameB = b.displayName.toUpperCase()
 
-  let comparison = 0
   if (nameA > nameB) {
-    comparison = 1
+    return 1
   } else if (nameA < nameB) {
-    comparison = -1
+    return -1
   }
-  return comparison
+  return 0
 }
```

Comment:
I think this can all be simplified to `return a.displayName.toUpperCase() > b.displayName.toUpperCase()`

Reply:
That doesn't handle the equality case right? Also, the Array sort function is expecting a number return value

Reply:
You are right. Looks like I've been playing fast and loose with returning boolean!